portfolio/src/main/webapp/gameoflife.js

```diff
@@ -16,7 +16,7 @@ resizeCanvasToDisplaySize(canvas);
 // Game of Life specific variables
 const SPEED = 200; // in miliseconds (speed between updates)
 const CELL_SIZE = 10; // in pixels (side length of individual cell)
-const SEED_SPAWN_RATE = 0.1; // as a ratio (likelihood a cell is 'alive' when seeded)
+const SEED_SPAWN_RATE = 0.05; // as a ratio (likelihood a cell is 'alive' when seeded)
```

Comment:
I think I remember this change from a previous PR.

I talked with Aleksey about this, and we think you'll need to run "git pull" first before rebasing onto master, since you'll need to pull the latest changes from your remote repo into your local repo before merging master.

Can you try doing this and see if that removes this diff? If not, we can try video chat tomorrow to resolve this.

Reply:
It said it was up to date. I think the issue was that I rebased instead of merged. Still not entirely sure tbh, we can talk more in gvc if you're free.

Reply:
As we talked about offline, let's try to reach out to a virtual support engineer to see what's going on. But if that doesn't help, then I think we'll have to live with the weird diffs